test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Csv.Tests.ps1

```diff
@@ -90,11 +90,25 @@ Describe "ConvertTo-Csv" -Tags "CI" {
         $result | Should -Not -Match ([regex]::Escape('#TYPE'))
     }
 
+    It "Does not support -UseQuotes and -QuoteFields at the same time" {
+        { $testObject | ConvertTo-Csv -UseQuotes Always -QuoteFields "TestFieldName" } |
+            Should -Throw -ErrorId "CannotSpecifyQuoteFieldsAndUseQuotes,Microsoft.PowerShell.Commands.ConvertToCsvCommand"
+    }
+
     It "Does not support -IncludeTypeInformation and -NoTypeInformation at the same time" {
         { $testObject | ConvertTo-Csv -IncludeTypeInformation -NoTypeInformation } |
             Should -Throw -ErrorId "CannotSpecifyIncludeTypeInformationAndNoTypeInformation,Microsoft.PowerShell.Commands.ConvertToCsvCommand"
     }
 
+    Context "QuoteFields parameter" {
```

Comment:
Seems like we should test multiple fields and also case insensitivity

Reply:
Done.